### ISSUE TITLE
[WASM + iPadOS] Uno.Gallery: various rendering issues

### ISSUE THREAD

**Issue by DierkDroth** [unoplatform/uno#10902]
### Current behavior

Please see screenshots below (iPadOS 16.2/Latest, Safari), there are various rendering issues
- in the navigation pane you will see large bubbles/circles which appear randomly as you clicked through the options. There even might be multiple of them at the same time. Is that expected?
- prior selections in the navigation pane still are highlighted. I don't think that's expected.

![download (3)](https://user-images.githubusercontent.com/26179242/210937654-cfd5e29a-a995-4391-a8e9-e10feb9f5b40.png)
![download](https://user-images.githubusercontent.com/26179242/210937661-b27cf4bd-5953-42ee-b199-3ad5bc7cccce.png)
![download (1)](https://user-images.githubusercontent.com/26179242/210937665-7ac2321d-59f3-42a6-acc7-32d17a2db0d4.png)
![download (2)](https://user-images.githubusercontent.com/26179242/210937667-0f4a24f9-53d6-4efa-8532-1fa1b2cda6fe.png)

### Works on UWP/WinUI

Yes

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia

### Affected platforms

iOS

**Comment by Youssef1313** [comment]
@DierkDroth Can you confirm if these are still issues?

**Comment by DierkDroth** [comment]
@Youssef1313 yep, I just clicked through the navigation pane on my iPad: UI still is screwed up. 
You should be able to reproduce yourself quite easily...

**Comment by MartinZikmund** [comment]
Applicable only to native renderers, works correctly in Skia targets
